eth/jobmanager.go

```diff
@@ -40,11 +40,16 @@ func (j *JobManager) Start() error {
 		for {
 			select {
 			case l := <-logsCh:
-				jid, strmID, _, tAddr, err := GetInfoFromJobEvent(l, j.c)
+				jid := new(big.Int).SetBytes(l.Data[0:32])
```

Comment:
The deserialization logic for log data might be different per event because `log.Data` will be a byte array of the non-indexed arguments of an event. We can probably include deserialization helpers for each event i.e. `ParseNewJobLog`